functions

```diff
@@ -147,3 +185,42 @@ function cleanUpTests() {
     cleanUpPath "$PARENT"
   fi
 }
+
+# Build a RFC date from passed params
+# Parameter order is 4 digit year, month, day, 24 clock hour, minute, seconds
+# Usage: buildRfcDate 2000 05 12
+# Usage: buildRfcDate 2000 05 12 17 32 45
+function buildRfcDate() {
+  if [ $# -ne 6 -a $# -ne 3 ]; then
+    echo "Invalid number of arguments expected 3 or 6"
+    exit 1
+  fi
+  local FMT="%Y%m%d%H%M%S"
+  local DATE="$1$2$3"
+  if [ $# -eq 6 ]; then
+    DATE="${DATE}$4$5$6"
+  else
+    DATE="${DATE}000000"
+  fi
+  echo $(date -jRuf "${FMT}" "${DATE}") 
```

Comment:
The `./version_tests.sh` script fails when calling `buildRfcDate` with error:
```
date: invalid option -- 'j'
```
Apparently the `-j` is not standard across operating systems.

Reply:
The `./version_tests.sh` does work all the way through if I provide hard-coded values for the `DATE` at:
https://github.com/whikloj/fcrepo4-tests/blob/c1900cfba4ea06344af18cbb19e64290060ca48f/version_tests.sh#L51
and
https://github.com/whikloj/fcrepo4-tests/blob/c1900cfba4ea06344af18cbb19e64290060ca48f/version_tests.sh#L154

Reply:
I'll play with this, I just liked not having to do all the formatting myself.